CONTRIBUTING.rst

```diff
@@ -44,14 +44,23 @@ Implementation Report Process
 As of early 2020, the process to generate an EARL report for the official
 `JSON-LD Processor Conformance`_ page is:
 
+* Check out a local copy of the test suites::
```

Comment:
I think it might be better to align with the main README test section for this: https://github.com/digitalbazaar/pyld#tests.  The opinionated setup there and in runtests.py is to checkout the test dirs as siblings of pyld.  runtests.py defaults to looking for those dirs so you don't have to list them on command line.  In the case of EARL report generation, the test dirs are explicit to avoid also running the normalization tests.  As is here you'd be instructed to check out dirs in different locations than when running regular tests.  Thoughts?

Reply:
Maybe this could just refer to the readme test section for setup?